css/_footer.scss

```diff
@@ -0,0 +1,151 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+/* Protocol overrides */
+
+// * -------------------------------------------------------------------------- */
```

Comment:
Most of this is pulled from bedrock - they wanted to use the same footer that is found on mozilla.org